Add the static values of rook ceph Configmap for OCS 4.9 in ocs-ci

The test `test_validate_ceph_config_values_in_rook_config_override` failed with the error:

> E       AssertionError: The Ceph config, set by rook-config-override is different than the expected. Please inform OCS-QE about this discrepancy. The expected values are:
> E         ['', '[global]', 'bdev_flock_retry = 20', 'mon_osd_full_ratio = .85', 'mon_osd_backfillfull_ratio = .8', 'mon_osd_nearfull_ratio = .75', 'mon_max_pg_per_osd = 600', 'mon_pg_warn_max_object_skew = 0', '[osd]', 'osd_memory_target_cgroup_limit_ratio = 0.5', '']
> E         The cluster's Ceph values are:['', '[global]', 'bdev_flock_retry = 20', 'mon_osd_full_ratio = .85', 'mon_osd_backfillfull_ratio = .8', 'mon_osd_nearfull_ratio = .75', 'mon_max_pg_per_osd = 600', 'mon_pg_warn_max_object_skew = 0', 'mon_data_avail_warn = 15', '[osd]', 'osd_memory_target_cgroup_limit_ratio = 0.5', '']
> E       assert Counter({'': ...io = 0.5': 1}) == Counter({'': ...io = 0.5': 1})
> E         Omitting 10 identical items, use -vv to show
> E         Left contains 1 more item:
> E         {'mon_data_avail_warn = 15': 1}

as you can see here https://ocs4-jenkins-csb-ocsqe.apps.ocp4.prod.psi.redhat.com/job/qe-deploy-ocs-cluster/7051/testReport/tests.manage.z_cluster.test_ceph_default_values_check/TestCephDefaultValuesCheck/test_validate_ceph_config_values_in_rook_config_override/ and here https://ocs4-jenkins-csb-ocsqe.apps.ocp4.prod.psi.redhat.com/job/qe-deploy-ocs-cluster/7106/testReport/tests.manage.z_cluster.test_ceph_default_values_check/TestCephDefaultValuesCheck/test_validate_ceph_config_values_in_rook_config_override/. From what I see the error occurred because we need to add the static values of rook ceph Configmap in the `constants` file https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/constants.py for OCS 4.9. 
Currently, we use these values in the line https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/constants.py#L1374: 

> ROOK_CEPH_CONFIG_VALUES = """
[global]
mon_osd_full_ratio = .85
mon_osd_backfillfull_ratio = .8
mon_osd_nearfull_ratio = .75
mon_max_pg_per_osd = 600
[osd]
osd_memory_target_cgroup_limit_ratio = 0.5
"""

--- a/ocs_ci/ocs/constants.py
+++ b/ocs_ci/ocs/constants.py
@@ -1395,6 +1395,20 @@ mon_pg_warn_max_object_skew = 0
 osd_memory_target_cgroup_limit_ratio = 0.5
 """
 
+ROOK_CEPH_CONFIG_VALUES_49 = """
+[global]
+bdev_flock_retry = 20
+mon_osd_full_ratio = .85
+mon_osd_backfillfull_ratio = .8
+mon_osd_nearfull_ratio = .75
+mon_max_pg_per_osd = 600
+mon_pg_warn_max_object_skew = 0
+mon_data_avail_warn = 15
+[osd]
+osd_memory_target_cgroup_limit_ratio = 0.5
+"""
+
+
 CEPH_DEBUG_CONFIG_VALUES = """
 [mon]
 debug_mon = 20

--- a/ocs_ci/ocs/resources/storage_cluster.py
+++ b/ocs_ci/ocs/resources/storage_cluster.py
@@ -741,10 +741,12 @@ def setup_ceph_debug():
         constants.CEPH_CONFIG_DEBUG_LOG_LEVEL_CONFIGMAP
     )
     ocs_version = version.get_semantic_ocs_version_from_config()
-    if ocs_version < version.VERSION_4_8:
-        stored_values = constants.ROOK_CEPH_CONFIG_VALUES.split("\n")
-    else:
+    if ocs_version == version.VERSION_4_8:
         stored_values = constants.ROOK_CEPH_CONFIG_VALUES_48.split("\n")
+    elif ocs_version >= version.VERSION_4_9:
+        stored_values = constants.ROOK_CEPH_CONFIG_VALUES_49.split("\n")
+    else:
+        stored_values = constants.ROOK_CEPH_CONFIG_VALUES.split("\n")
     ceph_debug_log_configmap_data["data"]["config"] = (
         stored_values + constants.CEPH_DEBUG_CONFIG_VALUES
     )

--- a/tests/manage/z_cluster/test_ceph_default_values_check.py
+++ b/tests/manage/z_cluster/test_ceph_default_values_check.py
@@ -2,8 +2,6 @@ import collections
 import logging
 import pytest
 
-from semantic_version import Version
-
 from ocs_ci.framework.pytest_customization.marks import bugzilla
 from ocs_ci.framework.testlib import (
     ManageTest,
@@ -17,6 +15,7 @@ from ocs_ci.framework import config
 from ocs_ci.ocs.ocp import OCP
 from ocs_ci.ocs import constants, defaults
 from ocs_ci.ocs.cluster import get_mds_cache_memory_limit
+from ocs_ci.utility import version
 
 
 log = logging.getLogger(__name__)
@@ -110,11 +109,13 @@ class TestCephDefaultValuesCheck(ManageTest):
             "Validating that the Ceph values, configured by ceph-config-override "
             "confiMap, match the ones stored in ocs-ci"
         )
-        ocs_version = config.ENV_DATA["ocs_version"]
-        if Version.coerce(ocs_version) < Version.coerce("4.8"):
-            stored_values = constants.ROOK_CEPH_CONFIG_VALUES.split("\n")
-        else:
+        ocs_version = version.get_semantic_ocs_version_from_config()
+        if ocs_version == version.VERSION_4_8:
             stored_values = constants.ROOK_CEPH_CONFIG_VALUES_48.split("\n")
+        elif ocs_version >= version.VERSION_4_9:
+            stored_values = constants.ROOK_CEPH_CONFIG_VALUES_49.split("\n")
+        else:
+            stored_values = constants.ROOK_CEPH_CONFIG_VALUES.split("\n")
         assert collections.Counter(config_data) == collections.Counter(stored_values), (
             f"The Ceph config, set by {constants.ROOK_CONFIG_OVERRIDE_CONFIGMAP} "
             f"is different than the expected. Please inform OCS-QE about this discrepancy. "

